trigger/src/main/scala/core/scheduler/utilities/Configs.scala

```diff
@@ -0,0 +1,67 @@
+package core.scheduler.utilities
+
+import java.util
+import java.util.{Collections, Properties}
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+private object Configs {
+  val conf: Config = ConfigFactory.load()
+}
+
+object KafkaConfig {
+  def getConsumerProperties: Properties = {
+    val properties = new Properties()
+    properties.put("bootstrap.servers", Configs.conf.getString("kafkaSource.bootstrap.servers"))
+    properties.put("group.id", Configs.conf.getString("kafkaSource.group.id"))
+    properties.put("key.deserializer", Configs.conf.getString("kafkaSource.key.deserializer"))
+    properties.put("value.deserializer", Configs.conf.getString("kafkaSource.value.deserializer"))
+    properties.put("max.poll.records", Configs.conf.getString("kafkaSource.max.poll.records"))
+    properties.put("enable.auto.commit", "false")
+    properties.put("auto.offset.reset", "latest")
```

Comment:
What happens in the first time?

Reply:
It will skip all messages. It is first implementation. It will be removed and user will have ability to set it up via UI.